#include <iostream>

int main()
{
    std::cout << "Running failing test" << std::endl;
    return 1;
}

